.github/workflows/publish.yml

```diff
@@ -22,32 +22,34 @@ jobs:
             ${{ runner.os }}-build-
             ${{ runner.os }}-
 
-      - name: Setup Node Environment ⬢
+      - name: Setup ⬢
         uses: actions/setup-node@v1
         with:
           node-version: 18
 
       - name: Tag 🏷️
-        uses: rickstaa/action-create-tag@v1
+        uses: actions/github-script@v5
+        id: create-tag
         with:
-          tag: ${{ github.event.inputs.release_version }}
-          message: "Release ${{ github.event.inputs.release_version }}"
+          github-token: ${{secrets.TEXTILEIO_MACHINE_ACCESS_TOKEN}}
+          script: |
+            await github.rest.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: 'refs/tags/${{ github.event.inputs.release_version }}',
+              sha: context.sha
+            })
 
-      - name: Update version ↗️
+      - name: Version 👍
         id: version-bump
         uses: jaywcjlove/github-action-package@v1.3.0
         with:
           version: ${{ github.event.inputs.release_version }}
 
-      - name: Setup 🐹
-        uses: acifani/setup-tinygo@v1
-        with:
-          tinygo-version: 0.23.0
```

Comment:
@carsonfarmer I don't know enough about how this works to say, but maybe the error has to do with this line?

Reply:
Ha, yes. See my comment above. I'll just update to 0.27 which I think is the latest. Having said that, it probably also means I need to update my local version, because the build is actually done locally right now.